multimodal-search-tirg/flow-query.yml

```diff
@@ -2,7 +2,6 @@
 with:
   prefetch: 10
   read_only: true  # better add this in the query time
-  # rest_api: true
   port_expose: $JINA_PORT
 pods:
```

Comment:
please update to `version 1` version of Flow

Reply:
done